src/domain/authentication/authService.ts

```diff
@@ -1,42 +1,79 @@
-import type { User, UserManagerSettings } from 'oidc-client';
-import { UserManager, Log, WebStorageStateStore } from 'oidc-client';
+import type { User, UserManagerSettings } from 'oidc-client-ts';
+import { UserManager, Log, WebStorageStateStore } from 'oidc-client-ts';
 import axios from 'axios';
 import * as Sentry from '@sentry/browser';
 
 import projectService from '../projects/projectService';
 import authorizationService from './authorizationService';
+import AppConfig from '../application/AppConfig';
 
 const origin = window.location.origin;
 export const API_TOKEN = 'apiToken';
 
+export type ApiTokenClientProps = {
+  url: string;
+  queryProps?: { permission: string; grant_type: string };
+  audiences?: string[];
+};
+
 export class AuthService {
   private userManager: UserManager;
+  private apiTokensClientConfig: ApiTokenClientProps;
+  private authServerType: 'KEYCLOAK' | 'TUNNISTAMO';
+  private audience: string;
 
   constructor() {
+    this.authServerType = AppConfig.oidcServerType ?? 'TUNNISTAMO';
```

Comment:
There's no need for the fallback value here as ´AppConfig.oidcServerType` always returns either `KEYCLOAK` or `TUNNISTAMO`.